### ISSUE TITLE
Using BrowserAnimationsModule causes applications to not unmount

### ISSUE THREAD

**Issue by joeldenning** [CanopyTax/single-spa-angular#48]
See related https://github.com/CanopyTax/single-spa/issues/289

<img width="352" alt="Screen Shot 2019-04-08 at 11 20 58 AM" src="https://user-images.githubusercontent.com/5524384/55743773-67341380-59f0-11e9-8eb7-4a7456f3aae7.png">

I have confirmed locally that the bug occurs.

**Comment by dassennato** [comment]
This issue is blocking my app of using material angular. I'd really need this fixed.

**Comment by joeldenning** [comment]
This seems like it might be related: https://github.com/angular/angular/issues/19742.

Still looking into this further

**Comment by joeldenning** [comment]
It turns out that this was completely unrelated to the issue I linked to previously. Instead, this is just a bug in BrowserAnimationsModule.

I implemented a workaround for the bug in https://github.com/CanopyTax/single-spa-angular/commit/d8159893bd5347822c810f420998cc0be94e4c84 and released it as 3.0.0-beta.14.

To get the fix, you need to provide singleSpaAngular the AnimationEngine option in your main.single-spa.ts:

```js
import { eAnimationEngine as AnimationEngine } from '@angular/animations/browser';

export default singleSpaAngular({
  ... // be sure to include the other opts you need.
  AnimationEngine,
})
```

Alternatively, you can let `ng add` do this for you, by answering "yes" to the question about whether you're using BrowserAnimationsModule.

See https://github.com/CanopyTax/single-spa-angular#single-spa-angular-options for more details
